handbook/03_building_something_real.md

```diff
@@ -56,7 +56,12 @@ export default class extends Controller {
 
 ### Common Actions Have a Shorthand Notation
 
-* (Describe why we can omit `click->` from the `data-action` attribute)
+`click->` can be omitted from the `data-action` attribute because it is one of default event names. Others are:
+- `a, button, input(type='submit')` => `click`
+- `form` => `submit`
+- `input, select, textarea` => `change`
+
+We can define other actions by explicitly defining event name, e.g: `paste->`.
```

Comment:
I like it. Maybe it would be clearer still to also show it in context, along the lines of:

> So instead of e.g. `<button data-action="click->clipboard#copy">`, you can write just `<button data-action="clipboard#copy">`.

Reply:
I made the changes like you suggest. Let me know if you want some others, or maybe squash commits into one.

Reply:
Thank you! To be clear, I'm not with Basecamp or have merge rights, so this was just a peer suggestion :) Fingers crossed for a merge, though.